integration-tests/pom.xml

```diff
@@ -133,6 +133,11 @@
             <slack.channel.name>mc</slack.channel.name>
 
             <webhook.site.uuid>${env.WEBHOOK_SITE_UUID}</webhook.site.uuid>
+
+            <ansible.endpoint>${env.ANSIBLE_ENDPOINT}</ansible.endpoint>
+            <ansible.username>${env.ANSIBLE_USERNAME}</ansible.username>
+            <ansible.password>${env.ANSIBLE_PASSWORD}</ansible.password>
+            <ansible.ssl.verification.disabled>true</ansible.ssl.verification.disabled>
```

Comment:
Are these documented in https://docs.google.com/spreadsheets/d/1Ud8BnqJJs6O0nZtD8o4WIFTqExcrfvofjCqWqJtX8YI/edit#gid=0 please?

Reply:
I've added the comment into the document. Hopefully it will be added soon.

Reply:
@jiripetrlik please see my respond in that google spreadsheet